src/main/kotlin/Winning.kt

```diff
@@ -0,0 +1,12 @@
+enum class Winning(val price: Int, val match: Int) {
+    FIRST(2000000000, 6),
+    SECOND(1500000, 5),
+    THIRD(50000, 4),
+    FOURTH(5000, 3);
+
+    companion object {
+        fun matchWinning(match: Int): Winning? {
```

Comment:
null을 반환하기 보다 1, 2, 3, 4등에 해당하지 않는 enum 케이스도 추가하면 어떨까요?
```suggestion
    FIRST(2000000000, 6),
    SECOND(1500000, 5),
    THIRD(50000, 4),
    FOURTH(5000, 3),
    NONE(0, 0)

    companion object {
        fun matchWinning(match: Int): Winning {
```